src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -140,16 +140,22 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
                 var checkerMessages = new List<ICheckerMessage>();
                 var scriptGenerator = new BuildScriptGenerator(serviceProvider, console, checkerMessages);
 
+                var generated = scriptGenerator.TryGenerateScript(out scriptContent);
+                stopwatch.AddProperty("generateSucceeded", generated.ToString());
+
                 if (checkerMessages.Count > 0)
                 {
                     var messageFormatter = new DefinitionListFormatter();
-                    checkerMessages.Select(msg => messageFormatter.AddDefinition(msg.Level.ToString(), msg.Content));
+                    checkerMessages.ForEach(msg => messageFormatter.AddDefinition(msg.Level.ToString(), msg.Content));
```

Comment:
Do we need tests for these lines of code?

Reply:
We can modify an existing end-to-end test to cover them